web-common/src/features/dashboards/workspace/DashboardHeader.svelte

```diff
@@ -69,8 +83,14 @@
             Edit this dashboard's metrics & settings
           </TooltipContent>
         </Tooltip>
-      </div>
-    {/if}
+      {/if}
+      <Tooltip distance={8}>
+        <Button on:click={openCalenderly} type="primary">Publish</Button>
```

Comment:
This should be behind the `isEditableDashboard` flag too. The read-only dashboards will be hosted/published already.